src/core/qgis.h

```diff
@@ -91,6 +91,58 @@ class CORE_EXPORT Qgis
     // Enumerations
     //
 
+    /**
+     * Authentication configuration storage capabilities.
+     *
+     * \since QGIS 3.40
+     */
+    enum class AuthConfigurationStorageCapability SIP_MONKEYPATCH_SCOPEENUM_UNNEST( QgsAuthConfigurationStorage, Capability ): int SIP_ENUM_BASETYPE( IntFlag )
```

Comment:
```suggestion
    enum class AuthConfigurationStorageCapability : int SIP_ENUM_BASETYPE( IntFlag )
```

We don't need that for newly added enums